<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderWindow</class>
 <widget class="QMainWindow" name="OrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>440</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:rgb(120, 180, 255);
color:white;
border-radius:3px;
}
QPushButton:hover{
border:none;
background-color:rgb(3, 180, 255);
color:white;
border-radius:3px;
}</string>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>333</width>
      <height>87</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>91</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>91</width>
         <height>25</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>91</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>邮发代号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Line1">
       <property name="minimumSize">
        <size>
         <width>331</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>331</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
border:1px solid black;
border-radius:3px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>请输入邮发代号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="signupinfo_1">
       <property name="maximumSize">
        <size>
         <width>111</width>
         <height>21</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 0, 0)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>333</width>
      <height>87</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="minimumSize">
        <size>
         <width>91</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>91</width>
         <height>25</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>91</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>期刊名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Line2">
       <property name="minimumSize">
        <size>
         <width>331</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>331</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
border:1px solid black;
border-radius:3px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>请输入期刊名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="signupinfo_2">
       <property name="maximumSize">
        <size>
         <width>111</width>
         <height>21</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 0, 0)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>333</width>
      <height>87</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="minimumSize">
        <size>
         <width>91</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>91</width>
         <height>25</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>91</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>订阅年份</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Line3">
       <property name="minimumSize">
        <size>
         <width>331</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>331</width>
         <height>41</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
border:1px solid black;
border-radius:3px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>请输入订阅年份</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="signupinfo_3">
       <property name="maximumSize">
        <size>
         <width>111</width>
         <height>21</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(255, 0, 0)</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:none;
background-color:rgb(174, 222, 255);
color:white;
border-radius:3px;
}
QPushButton:hover{
border:none;
background-color:rgb(3, 180, 255);
color:white;
border-radius:3px;
}</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="MoveLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:rgb(0, 119, 239);
color:white;
}</string>
    </property>
    <property name="text">
     <string>新增征订</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>441</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>layoutWidget_2</zorder>
   <zorder>layoutWidget_3</zorder>
   <zorder>layoutWidget_4</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label</zorder>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MoveLabel</class>
   <extends>QLabel</extends>
   <header location="global">movelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
